.github/workflows/test.yml

```
@@ -0,0 +1,23 @@
+  push:
+    branches: [ main ]
+  pull_request:
+    types: [ opened, synchronize ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v4
+    
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)